static/app/views/issueList/header.tsx

```diff
@@ -96,86 +96,81 @@ function IssueListHeader({
-          </ButtonBar>
-        </Layout.HeaderActions>
-        <StyledGlobalEventProcessingAlert projects={selectedProjects} />
-      </BorderlessHeader>
-      <TabLayoutHeader>
-        <Layout.HeaderNavTabs underlined>
-          {visibleTabs.map(
-            ([tabQuery, {name: queryName, tooltipTitle, tooltipHoverable}]) => {
-              const to = {
-                query: {
-                  ...queryParms,
-                  query: tabQuery,
-                  sort:
-                    tabQuery === Query.FOR_REVIEW ? IssueSortOptions.INBOX : sortParam,
-                },
-                pathname: `/organizations/${organization.slug}/issues/`,
-              };
-
-              return (
-                <li key={tabQuery} className={query === tabQuery ? 'active' : ''}>
-                  <Link to={to} onClick={() => trackTabClick(tabQuery)}>
-                    <WrapGuideTabs query={query} tabQuery={tabQuery} to={to}>
-                      <Tooltip
-                        title={tooltipTitle}
-                        position="bottom"
-                        isHoverable={tooltipHoverable}
-                        delay={1000}
-                      >
-                        {queryName}{' '}
-                        {queryCounts[tabQuery]?.count > 0 && (
-                          <Badge
-                            type={
-                              tabQuery === Query.FOR_REVIEW &&
-                              queryCounts[tabQuery]!.count > 0
-                                ? 'review'
-                                : 'default'
-                            }
-                          >
-                            <QueryCount
-                              hideParens
-                              count={queryCounts[tabQuery].count}
-                              max={queryCounts[tabQuery].hasMore ? TAB_MAX_COUNT : 1000}
-                            />
-                          </Badge>
-                        )}
-                      </Tooltip>
-                    </WrapGuideTabs>
-                  </Link>
-                </li>
-              );
-            }
-          )}
-          <SavedSearchTab
-            organization={organization}
-            query={query}
-            sort={sort}
-            savedSearchList={savedSearchList}
-            onSavedSearchSelect={onSavedSearchSelect}
-            onSavedSearchDelete={onSavedSearchDelete}
-            isActive={savedSearchTabActive}
-            queryCount={queryCount}
-          />
-        </Layout.HeaderNavTabs>
-      </TabLayoutHeader>
-    </React.Fragment>
+    <Layout.Header noActionWrap>
+      <Layout.HeaderContent>
+        <StyledLayoutTitle>{t('Issues')}</StyledLayoutTitle>
+      </Layout.HeaderContent>
+      <Layout.HeaderActions>
+        <ButtonBar gap={1}>
+          <Button
+            size="small"
+            data-test-id="real-time"
+            title={t('%s real-time updates', realtimeActive ? t('Pause') : t('Enable'))}
```

Comment:
wouldn't it be better if we had the entire translation in one sentence? maybe the current form is not ideal when translating the sentence into other languages 🤷‍♀️

```suggestion
            title={realtimeActive ? t('Pause real-time updates') : t('Enable real-time updates')}
```

Reply:
Yeah probably. I'll fix in a follow up since I just copied this over.